app/assets/javascripts/cable.js

```diff
@@ -0,0 +1,13 @@
+// Action Cable provides the framework to deal with WebSockets in Rails.
+// You can generate new channels where WebSocket features live using the rails generate channel command.
+//
+//= require action_cable
+//= require_self
+//= require_tree ./channels
+
+(function() {
```

Comment:
Per action cable's docs, you might need to add `<%= action_cable_meta_tag %>` to the applicaiton.html.erb layout: see https://github.com/rails/rails/tree/master/actioncable#consumer-configuration

Reply:
this is to separate configurations between local and production, really not familiar with the production side of things but i could give it a shot.